grafana-plugin/src/components/IntegrationContactPoint/IntegrationContactPoint.tsx

```diff
@@ -250,6 +250,17 @@ export const IntegrationContactPoint: React.FC<{
   }
 
   function renderActions(item: ContactPoint) {
+    const onDisconnect = async () => {
+      try {
+        await AlertReceiveChannelHelper.disconnectContactPoint(id, item.dataSourceId, item.contactPoint);
+        closeDrawer();
+        openNotification('Contact point has been removed');
+        alertReceiveChannelStore.fetchConnectedContactPoints(id);
+      } catch (_err) {
+        openErrorNotification('An error has occurred. Please try again.');
```

Comment:
Add please this generic in `consts.ts` (I did the same in my PR) so that we reuse this error

`export const GENERIC_ERROR = 'An error has occurred. Please try again';`

Reply:
Thanks, I've done it now for all the places 👍